server/utils/firebase_algolia_server_code/functions/test/index.test.js

```diff
@@ -0,0 +1,113 @@
+/**
+ * Description: Unit tests for index.js
+ *              Uses dependency injection to mock algolia save/delete functions
+ *              Uses sinon/chai to spy on calls and validate behavior
+ *              compile with `npm run test`
+ * Author: Robert Marcus
+ * Date: July 7, 2020
+ */
+
+// Import the testing environment configuration
+const { test, assert, sinon } = require('./testConfig');
+
+// Import the generic cloud functions.
+const { createRecordFromEntity, updateRecord, deleteRecord } = 
+    require('../index.js');
+
+// Import mock saveObject, mock deleteObject, and helpers.
+const { saveObject, deleteObject } = require('./algoliaMocks');
+const { getExpectedOutputID, getExpectedOutputSnap, getExpectedOutputChange } =
+    require('./expectedMockOutputHelpers');
+
+describe("test_createRecordFromEntity", () => {
+  it('should call saveObject exactly once', () => {
+    const saveSpy = sinon.spy(saveObject);
+    const snap = test.firestore.exampleDocumentSnapshot();
+
+    const createRecordWrapper = test.wrap(createRecordFromEntity(saveSpy));
+    const addedSnap = createRecordWrapper(snap);
```

Comment:
No need to store the return value `addedSnap` since you don't use it here. Same below.

Reply:
Why was this marked as resolved?

Reply:
True in general for Aileme's comments in this file.  I have most of the same comments and a lot of them are marked resolved and they don't look resolved to me.

Reply:
Moot.